deploy/helm/sumologic/values.yaml

```diff
@@ -4302,6 +4302,34 @@ otellogs:
     #     mountPath: /certs
     #     readOnly: true
 
+  ## additionalDaemonSets allows to set daemonsets with affinity, nodeSelector and resources
+  ## different than the main DaemonSet
+  ## Be careful and set nodeAffinity for the main DaemonSet,
+  ## as we do not support multiple pods of otellogs on the same node
+  ##
+  ## e.g:
+  ## additionalDaemonSets:
+  ##   - nodeSelector:
+  ##       kubernetes.io/os: linux
+  ##     resources:
+  ##       limits:
+  ##         memory: 1Gi
+  ##         cpu: 6
+  ##       requests:
+  ##         memory: 32Mi
+  ##         cpu: 6
+  ## daemonset:
+  ##   affinity:
+  ##     nodeAffinity:
+  ##       requiredDuringSchedulingIgnoredDuringExecution:
+  ##         nodeSelectorTerms:
+  ##         - matchExpressions:
+  ##           - key: kubernetes.io/os
+  ##             operator: NotIn
+  ##             values:
+  ##             - linux
+  additionalDaemonSets: []
```

Comment:
How about a map of named daemonsets? This might make it easier to use this feature, having a daemonset named e.g. `otelcol-logs-collector-hpc` instead of `otelcol-logs-collector-1`. Just a suggestion, it has its ins and outs (like making the names too long).

Reply:
this is actually a good idea. I will work on it